### DISCUSSION TITLE
502 Bad gateway error

### DISCUSSION THREAD

**Discussion by HamzaTehseen** [airbytehq/airbyte#41667] [General]
Hi, I have been trying to run airbyte since yesterday on my Windows 11 machine and facing the same error. Upon running the container and the `run-ab-platform.sh` command on cmd, when I open _localhost:8000_, I receive the folwing error:

`{
  "url": "http://localhost:8000/",
  "airbyteVersion": "0.63.6",
  "errorType": "HttpError",
  "errorConstructor": "Ch",
  "error": {
    "i18nKey": "errors.http.badGateway",
    "i18nParams": {
      "status": 502
    },
    "name": "HttpError",
    "requestId": "mFziwx3MH3rs9uJ7xzSQiF",
    "request": {
      "url": "/api/v1/instance_configuration",
      "method": "get"
    },
    "status": 502,
    "response": "<!DOCTYPE html>\n<html>\n<head>\n<title>Error</title>\n<style>\nhtml { color-scheme: light dark; }\nbody { width: 35em; margin: 0 auto;\nfont-family: Tahoma, Verdana, Arial, sans-serif; }\n</style>\n</head>\n<body>\n<h1>An error occurred.</h1>\n<p>Sorry, the page you are looking for is currently unavailable.<br/>\nPlease try again later.</p>\n<p>If you are the system administrator of this resource then you should check\nthe error log for details.</p>\n<p><em>Faithfully yours, nginx.</em></p>\n</body>\n</html>\n"
  },
  "stacktrace": "HttpError: errors.http.badGateway\n    at Fpe (http://localhost:8000/assets/core-cvfvb7yyld.js:119:9674)",
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36",
  "featureFlags": {}
}`

and in my docker container, I receive the following:

`request returned Internal Server Error for API route and version http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.45/containers/json?all=1&filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.oneoff%3DFalse%22%3Atrue%2C%22com.docker.compose.project%3Dairbyte%22%3Atrue%7D%7D, check if the server supports the requested API version`

I can't find solution to this problem and it's urgent. Please help as soon as possible. Thank You
![Screenshot (1397)](https://github.com/user-attachments/assets/46c5dfc0-c8d7-40df-84e0-95cd2593f49a)
![Screenshot (1398)](https://github.com/user-attachments/assets/0f59f4b4-7e7c-4178-99de-a8e2b6d7b56b)
![Screenshot (1399)](https://github.com/user-attachments/assets/3ba82366-0da9-4f4b-8569-9fea5dd33916)

**Comment by Sakiruto** [comment]
@HamzaTehseen Did you resolve this issue , if yes how did you do it ??

@visnup @mid can you guys help us with the issue ?!

[a comment by an automated account is not shown]
